pkg/api/common/versions.go

```diff
@@ -214,9 +219,9 @@ func GetSupportedVersions(orchType string, hasWindows bool) (versions []string,
 	switch orchType {
 	case Kubernetes:
 		if hasWindows {
-			return GetAllSupportedKubernetesVersionsWindows(), string(KubernetesDefaultVersion)
+			return GetAllSupportedKubernetesVersionsWindows(), GetDefaultKubernetesVersion()
 		}
-		return GetAllSupportedKubernetesVersions(), string(KubernetesDefaultVersion)
```

Comment:
Any reason why a string conversion was necessary here? I removed it but let me know if I'm missing something @jackfrancis

Reply:
not a sane one that I can think of, nope!